src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.ArrayList;
+
+/**
+ * This class is the main point of entry for this project.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private final TaskList taskList = new TaskList();
+    private final Storage storage = new Storage();
+
+    public Duke() {
+        this.initialise();
+    }
+
+    public String greet() {
+        return "Hello, I am Duke, your personal assistant.";
+    }
+
+    /**
+     * Initialises Duke by reading the stored data from disk and storing the tasks
+     * in {@code TaskList}.
+     */
+    private void initialise() {
```

Comment:
Maybe you can handle commands in another class to make your make Duke class looks neater?